rust/nft-candy-machine/src/lib.rs

```diff
@@ -505,6 +523,7 @@ pub struct CandyMachine {
     pub authority: Pubkey,
     pub wallet: Pubkey,
     pub token_mint: Option<Pubkey>,
+    pub secondary_token_mint: Option<Pubkey>,
```

Comment:
there's more ramifications for adding additional struct fields, will need to update the CLI / docs / figure out how this should actually be configured. the instructions above don't require these changes, though